designs/Makefile

```diff
@@ -0,0 +1,31 @@
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+
+SOURCES := $(shell find ${ROOT_DIR} -name \*.puml)
+DIAGRAMS := $(SOURCES:%.puml=%.png)
+
+REGISTRY ?= public.ecr.aws/i0f3w2d9
```

Comment:
So is this a shared registry just for this? Should we create an official one to use from build tooling?

Reply:
This is my own account
I don't think we should be pushing manually to a build account and I didn't think it was worth to setup post submit infrastructure to automate this, given that this image will rarely change.

Reply:
True but I think the main benefit is that we can have an official place that this image is stored rather than your account, otherwise it will end up becoming hard to figure out what account this is from when people want to actually pull the image.

Reply:
Does it matter what account this is in to pull the image?
Imagine this image was already available in docker.io, would that make the pull process more difficult?

Building automation for this image seems complete overkill imo. But I think the way this works right now allows to do that without mayor changes here. Basically just change the registry url.